types/node-forge/index.d.ts

```diff
@@ -14,7 +14,7 @@
 // TypeScript Version: 2.6
 
 /// <reference types="node" />
-
+import * as jsbn from "jsbn";
```

Comment:
We need to add a dependency on `@types/jsbn` right?

https://github.com/DefinitelyTyped/DefinitelyTyped#i-notice-some-packages-having-a-packagejson-here